docs/application/tizen-studio/release-notes/3-3-release-notes.md

```diff
@@ -0,0 +1,67 @@
+# Tizen Studio 3.3 Release Notes
+
+-   Release Date: July 26, 2019
+
+
+## IDE and Tools
+
+
+### Fixed Bugs
+-   Device Manager
+	-   The issue of logs not being displayed for the Raspberry Pi board has been fixed.
+-   IDE
+	-   The issue of the log view for TV devices in the Tizen IDE has been fixed.
+	-   The issue of the app signing command incorrectly asking for the author and the distributor passwords has been fixed.
+	-   Running an application on the emulator now launches the emulator automatically if the emulator is not in the launched state.
+	-   The launch_screen element to platform schema for the web applications has been added.
```

Comment:
One more question, "launch_screen" element is part of the code?
If it is, then can we use the code tag to emphasize?

The `launch_screen` element...

Reply:
it is part of schema.